hycom/RELO/config/Linux.fram.ifort_hycom

```diff
@@ -15,25 +15,23 @@
 # LDFLAGS:    Loader flags.
 # EXTRALIBS:  Extra local libraries (if any).
 #
-#INCLUDES      = -I/cluster/software/netCDF-Fortran/4.4.4-intel-2017a-HDF5-1.8.18/include  -I${ESMF_MOD_DIR} -I${ESMF_DIR}include -I./CICE/rundir/compile
-
-# --- Mostafa: flags required for standalone HYCOM
-INCLUDES      = -I${ESMF_MOD_DIR} -I${ESMF_DIR}include -I./CICE/rundir/compile
+OASIS_DIR     = /cluster/projects/nn9878k/guibou/NANUK/oa3-mct_iimpi2022a/
+LIBPSMILE     = $(OASIS_DIR)/lib/libpsmile.MPI1.a $(OASIS_DIR)/lib/libmct.a $(OASIS_DIR)/lib/libmpeu.a $(OASIS_DIR)/lib/libscrip.
+a
```

Comment:
The code seems to compile of with the extension "a" of "libscrip" on the next line (line 20) rather than at the end of $(OASIS_DIR)/lib/libscrip. , but it would be best to delete the newline command for clarity